src/main/java/de/uhd/ifi/se/decision/management/jira/webhook/WebhookContentProvider.java

```diff
@@ -71,9 +101,83 @@ private String createWebhookData() {
 		return "{\"issueKey\": \"" + this.rootElementKey + "\", \"ConDecTree\": " + treantAsJson + "}";
 	}
 
+	private String createWebhookDataForSlack(KnowledgeElement changedElement) {
+		if(changedElement == null|| changedElement.getSummary() == null|| changedElement.getType() == null || changedElement.getUrl() == null){
+			return "";
+		}
+		String summary = knowledgeElement.getSummary();
+		System.out.println("Summary is now: "+summary);
+
+		if( summary.contains("{")){
+			System.out.println("Summary contains {.");
+			summary = cutSummary(changedElement.getSummary());
+			System.out.println("Summary is now: "+summary);
+		}
+		String data = "{'blocks':[{'type':'section','text':{'type':'mrkdwn','text':'Folgendes Entscheidungswissen wurde angepasst'}},"+
+		"{'type':'section','text':{'type':'mrkdwn','text':'*Typ:* :"+ changedElement.getType() + ": : " + changedElement.getType() +
+		" \\n *Titel*: " + summary + "\\n'},"+
+		"'accessory':{'type':'button','text':{'type':'plain_text','text':'Go to Jira'},'url' : '"+changedElement.getUrl()+"'}}]}";
+		System.out.println("createPostMethod: data"+data);
+
+		return data;
+	}
+
+
+		public PostMethod createPostMethodForSlack(){
+			if(knowledgeElement == null ){
+				return new PostMethod();
+			}
+			return createPostMethodForSlack(this.knowledgeElement);
+		}
+	/**
+	 * Creates the key value JSON String transmitted via webhook for Slack.
+	 *(Differences: "text", mask \ and ")
+	 * @return JSON String with the following pattern: {"text": " \"issueKey\": {String},
+	 *         \"ConDecTree\": {TreantJS JSON config and data} " }
+	 */
+	public PostMethod createPostMethodForSlack(KnowledgeElement changedElement) {
+		PostMethod postMethod = new PostMethod();
+		if (projectKey == null || changedElement == null || receiver == null) {
+				System.out.println("createPostMethod null");
+			return postMethod;
+		}
+		String webhookData = "";
+		if(receiver == "Other"){
+			LOGGER.info("receiver:  Other");
+			System.out.println("createPostMethodForSlack:receiver= other");
+			return createPostMethod();
+
+		}
+		if(receiver == "Slack"){
+			LOGGER.info("receiver:  Slack");
+			//System.out.println("createPostMethodForSlack:receiver= slack");
+			webhookData = createWebhookDataForSlack(changedElement);
+			//System.out.println(webhookData);
+		}
+		if(webhookData == "" || webhookData == null){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5269053)